examples_utils/src/camera.rs

```diff
@@ -54,6 +85,16 @@ impl Default for CameraController {
     }
 }
 
+pub fn print_controls(controller: Query<&CameraController>, mut controls_printed: Local<bool>) {
```

Comment:
Could this use change detection instead?

Reply:
Do you mean to detect if controls have been changed after the initial creation of the `CameraController`? Currently it detects when the camera with the controller is initiated and prints controls of the controller once.

Reply:
Yes, in your other comment you mentioned wanting to support changing controls. This way if the controls changed during runtime you would see the new controls in the console, but I guess this is fine for now since we don't need that right now.